apps/condo/domains/banking/schema/BankContractorAccount.js

```diff
@@ -0,0 +1,92 @@
+/**
+ * Generated by `createschema banking.BankContractorAccount 'name:Text; organization:Relationship:Organization:CASCADE; costItem?:Relationship:BankCostItem:SET_NULL; tin:Text; country:Text; routingNumber:Text; number:Text; currencyCode:Text; importId?:Text; territoryCode?:Text; bankName?:Text; meta?:Json; tinMeta?:Json; routingNumberMeta?:Json'`
+ */
+
+const { Json } = require('@open-condo/keystone/fields')
+const { Relationship, Text } = require('@keystonejs/fields')
+const { GQLListSchema } = require('@open-condo/keystone/schema')
+const { historical, versioned, uuided, tracked, softDeleted, dvAndSender } = require('@open-condo/keystone/plugins')
+const access = require('@condo/domains/banking/access/BankContractorAccount')
+const { ORGANIZATION_OWNED_FIELD } = require('@condo/domains/organization/schema/fields')
+const { CURRENCY_CODE_FIELD, IMPORT_ID_FIELD } = require('../../common/schema/fields')
+
+
+const BankContractorAccount = new GQLListSchema('BankContractorAccount', {
+    schemaDoc: 'Account for external contractor, which organization is not stored in condo. Contractor is related to expenses transactions of an Organization. Will be obtained from export files with bank transactions or from banking API',
+    fields: {
+
+        name: {
+            schemaDoc: 'Name of contractor organization',
+            type: Text,
+            isRequired: true,
+        },
+
+        organization: {
+            ...ORGANIZATION_OWNED_FIELD,
+            schemaDoc: 'Technical link to property management organization, that has expenses transactions with this contractor. Used for access rights and cascade behaviour of data. Contractor organization is not stored in condo. The object will be deleted if the related organization ceases to exist',
+        },
+
+        costItem: {
+            schemaDoc: 'Related cost item, that will determine cost items for all transactions belonging to this contractor',
```

Comment:
All transactions where the `costItem` is not defined

Reply:
Really not all. Every transaction without the `costItem` field

Reply:
Changed